cuenca/jwt.py

```diff
@@ -0,0 +1,47 @@
+import base64
+import binascii
+import datetime as dt
+import json
+import typing
+from dataclasses import dataclass
+
+from .exc import MalformedJwtToken
+
+if typing.TYPE_CHECKING:
+    from .http import Session  # pragma: no cover
```

Comment:
Use a config for this instead: https://coverage.readthedocs.io/en/stable/excluding.html

Adding an exclusion for this particular `if` statement is good practice across all projects. iirc, Pydantic does something similar

Reply:
Done, using the config was way better.